docs/guide/blog/comment-blog.md

```diff
@@ -175,7 +175,7 @@ func (k msgServer) CreateComment(goCtx context.Context, msg *types.MsgCreateComm
 	}
 
 	// Check if the Post Exists for which a comment is being created
-	if msg.PostID > postId && msg.PostID == postId {
+	if msg.PostID != postId {
```

Comment:
```suggestion
	if msg.PostID == 0 {
```

Shouldn't we do something like this, and also move this check to line 166?

Reply:
my mistake. suggestion looks great!